### PR TITLE
add aleph perl profile

### PR DESCRIPTION
**Pull request by umjnega**

### CODE REVIEW

File: manifests/profile/aleph/perl.pp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# nebula::profile::aleph::perl
+#
+# Install perl dependencies for aleph/makersmark applications
+#
+# @example
+#   include nebula::profile::aleph::perl
+class nebula::profile::aleph::perl () {
+
+  ensure_packages([
```
Comment (botimer) [inline manifests/profile/aleph/perl.pp:14]: Sort alpha

File: manifests/profile/aleph/perl.pp [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    'libspreadsheet-parseexcel-perl',
+    'libspreadsheet-writeexcel-perl',
+    'libspreadsheet-xlsx-perl',
+  ])
+
+  nebula::cpan { [
+    'MARC::File::XML',
+    'SMS::Send::Twilio']:
+  }
+
```
Comment (botimer) [inline manifests/profile/aleph/perl.pp:29]: Remove

**Review by botimer — CHANGES_REQUESTED** [review]
Couple of minor textual items. Otherwise, looks great.

File: manifests/profile/aleph/perl.pp (hunk 1 above)
Comment (umjnega) [reply manifests/profile/aleph/perl.pp:14]: Not sure what you mean by this one?

Comment (umjnega) [reply manifests/profile/aleph/perl.pp:13]: Sorted alphabetically.   Should be all set.

**Review by botimer — APPROVED** [review]

**Pull request merged**